docs/plugins/transform-decorators.md

```diff
@@ -21,3 +21,7 @@ Add the following line to your `.babelrc` file:
   "plugins": ["transform-decorators"]
 }
 ```
+
+## References
+
+* [Javascript Decorators](https://github.com/wycats/javascript-decorators/blob/master/README.md)
```

Comment:
Can you prefix these proposals with `Proposal:`

In general it'd be good to say where they are from first (ie. `MDN: Classes`, `Spec: Classes`, `Proposal: Decorators` etc.)